From: Josep Puigdemont <josep.puigdemont@enea.com>
Date: Wed, 4 May 2016 14:27:23 +0200
Subject: [PATCH] deploy.sh: do not expect a parameter for -h

If -h was given as a parameter to the script, it would report an error
as it expected a parameter, and if it was called as the only parameter,
it would run deploy.py as if "old style" parameters had been given, thus
showing the usage for the python script, instead of the expected usage
message for this script.

Update the usage message to include -h.

Change-Id: I0930936962c1cb479ec4409ff114cd60a386b276
Signed-off-by: Josep Puigdemont <josep.puigdemont@enea.com>
---
 ci/deploy.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/ci/deploy.sh b/ci/deploy.sh
index d83bba2..dc13f1c 100755
--- a/ci/deploy.sh
+++ b/ci/deploy.sh
@@ -40,6 +40,7 @@ OPTIONS:
   -f  Deploy on existing Fuel master
   -e  Do not launch environment deployment
   -F  Do only create a Fuel master
+  -h  Print this message and exit
   -H  No health check
   -l  Lab-name
   -p  Pod-name
@@ -62,6 +63,7 @@ Input parameters to the build script is:
 -f Deploy on existing Fuel master
 -e Do not launch environment deployment
 -F Do only create a Fuel master
+-h Print this message and exit
 -H Do not run fuel built in health-check after successfull deployment
 -l Lab name as defined in the configuration directory, e.g. lf
 -p POD name as defined in the configuration directory, e.g. pod-1
@@ -116,7 +118,7 @@ DRY_RUN=0
 ############################################################################
 # BEGIN of main
 #
-while getopts "b:B:dfFHl:p:s:S:i:h:e" OPTION
+while getopts "b:B:dfFHl:p:s:S:i:he" OPTION
 do
     case $OPTION in
         b)
